.vsts.pipelines/jobs/ci-linux.yml

```diff
@@ -66,6 +67,7 @@ jobs:
       $(docker.run) $(docker.src.map) $(docker.src.work) $(imageName) ./build.sh \
         /p:Configuration=$(sb.configuration) \
         /p:PortableBuild=$(sb.portable) \
+        /p:UseSystemLibunwind=$(systemLibunwind) \
```

Comment:
If I'm not mistaken you'll need this in the tarball build too, or else the smoke tests will fail:

https://github.com/dotnet/source-build/blob/169b6db68233ff598aad330f9a5217f62e482fab/.vsts.pipelines/jobs/ci-linux.yml#L206-L210

Reply:
Or, maybe it'll fail to build when it tries to link against the system lib that doesn't exist? I don't quite know where we run into trouble here.

Reply:
Thanks, fixed.